docs/docs/userguide/PROMETHEUS.md

```diff
@@ -1,11 +1,19 @@
-# Prometheus Monitoring
+# Monitoring
 
-## Prometheus Operator
+The instructions outlined on this page provide details on how you can enable [Prometheus](https://prometheus.io/){.external} monitoring on your stack with [Grafana](https://grafana.com/){.external}
 
-Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) Helm chart (call your release `prometheus-stack`).
+## 1. Install Prometheus Operator
 
-Once [JMX metrics are exposed](../../../userguide/OPERATION/#monitoring) in your product Helm chart, Helm will create a dedicated `JMX` service, and you can now create a `ServiceMonitor` object so that Prometheus starts scraping metrics:
+Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack){.external} Helm chart (call your release `prometheus-stack`).
 
+## 2. Expose JMX metrics
+
+Enable and expose [JMX metrics](../../../userguide/OPERATION/#expose-jmx-metrics) in your product Helm chart, Helm will create a dedicated `JMX` service. 
+
+
+## 3. Create a ServiceMonitor
+
+Now that `JMX` metrics are exposed, we need a way of scraping them. Using the `yaml` below create a `ServiceMonitor` object so that Prometheus cam scraping the exposed JMX metrics
```

Comment:
```suggestion
Now that `JMX` metrics are exposed, we need a way of scraping them. Using the `yaml` below create a `ServiceMonitor` object so that Prometheus starts scraping the exposed JMX metrics
```